src/components/SelectionList/TextWithIconCell.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import Text from '@components/Text';
+import Tooltip from '@components/Tooltip';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type TextWithIconCellProps = {
+    icon: IconAsset;
+    text?: string;
+    showTooltip: boolean;
+};
+
+export default function TextWithIconCell({icon, text, showTooltip}: TextWithIconCellProps) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    if (!text) {
+        return null;
+    }
+
+    return (
+        <Tooltip
+            shouldRender={showTooltip}
+            text={text}
+        >
+            <View style={[styles.flexRow, styles.flexShrink1, styles.gap1]}>
```

Comment:
If we use Tooltip here, the tooltip will show even when the text is not truncated, which causes https://github.com/Expensify/App/issues/68212

We fixed it here:
https://github.com/Expensify/App/pull/68849